app/jobs/build_submission_bundle_job.rb

```diff
@@ -1,14 +1,14 @@
 class BuildSubmissionBundleJob < ApplicationJob
   def perform(submission_id)
     submission = EfileSubmission.includes(:intake, :qualifying_dependents, :address, :tax_return, client: :efile_security_informations).find(submission_id)
-    persist_dependent_records(submission)
```

Comment:
Creating the dependents has moved into the 'preparing' step in the efile submission state machine.